### ISSUE TITLE
Defrag Wallet error

### ISSUE THREAD

**Issue by git70** [siacentral/sia-lite-wallet-web#10]
Hi

When trying to send the transfer, I got the message: "Transaction has too many inputs. Wallet needs to be defragmented before sending."

Then, when trying to defragment, the message: "POST request error; error accepting transaction set: transaction set is too large for this transaction pool"

Please help!

**Comment by n8maninger** [comment]
That must be a very large wallet.. How many outputs and transactions are needed for defrag? I believe this issue has been fixed already, can you try a hard refresh with `CMD+SHIFT+R` or `CTRL+F5` to make sure you're not caching an old version?

**Comment by git70** [comment]
> How many outputs and transactions are needed for defrag?

How can I check it?
 
> To make sure you're not caching an old version can you try a hard refresh with `CMD+SHIFT+R` or `CTRL+F5`? Let me know if that fixes your issue.

My browser cache is clean and still not working :(

**Comment by n8maninger** [comment]
It should say how many outputs and transactions need defragmenting right on the modal

![Screen Shot 2020-09-28 at 08 18 45](https://user-images.githubusercontent.com/5544150/94378288-440c9f00-0163-11eb-814c-25ad2620298a.png)

**Comment by git70** [comment]
It doesn't show it. Immediately after entering the amount, it shows "Transaction has too many inputs. Wallet needs to be defragmented before sending"

**Comment by git70** [comment]
Before starting defragmentation, it showed:
# of Outputs (Before) 443
# of Transactions 5

**Comment by git70** [comment]
Have you fixed something yet?
Now the defragmentation has gone. I guess ...

**Comment by n8maninger** [comment]
Are you able to send transactions okay now? If the defag button is no longer visible, then your wallet should be okay to spend your balance. Sorry for any annoyance..

I did push a new version, but it adds a version identifier to the unlock page. It did not change the defrag code. I still believe the issue you ran into was fixed by c6bb76e which broadcasts each defrag transaction one at a time instead of in a single transaction set.
![Screen Shot 2020-09-28 at 08 48 49](https://user-images.githubusercontent.com/5544150/94378853-76200000-0167-11eb-95da-66014a5b0808.png)
